plugin/cache/README.md

```diff
@@ -69,6 +70,10 @@ cache [TTL] [ZONES...] {
   greater than 5 minutes.
 * `disable`  disable the success or denial cache for the listed **ZONES**.  If no **ZONES** are given, the specified
   cache will be disabled for all zones.
+* `keepttl` enforce the cache plugin to not reduce the TTL he got from its backend. This can be useful
+  if CoreDNS is used as an authoritative server and you want to serve a consistent TTL to the clients.
+  This is **NOT** recommended when CoreDNS is not an authoritative server as using this option could result
```

Comment:
```suggestion
  This is **NOT** recommended when CoreDNS is caching records it is not authoritative for because it could result
```

Reply:
Suggesting this because CoreDNS can be hybrid authoritative for some zones, and non-authoritative others.